src/main/java/frc/robot/Constants.java

```diff
@@ -52,6 +52,12 @@ public static final class NeoMotorConstants {
   public static final class IntakeConstants {
     public static final double rotationSpeed = 0.5;
     public static final double rollerSpeed = 0.5;
+    public static final double kP = 0.0;
+    public static final double kI = 0.0;
+    public static final double kD = 0.0;
+    public static final double kS = 0.0;
+    public static final double kV = 0.0;
+    public static final double VELOCITY_CONVERSION_FACTOR = 0.0;
   }
 
   public static final class PIDConstants {
```

Comment:
what are these pid constants for?

Reply:
also lines 69-70 (rotliftsetpoint and rot lower setpoint, move those into a class of its own!

(then u can get rid of the rotation label)